sentinel-2/kndvi/README.md

```diff
@@ -0,0 +1,53 @@
+---
+title: kernel NDVI
+parent: Sentinel-2
+grand_parent: Sentinel
+layout: script
+permalink: /sentinel-2/kndvi/
+nav_exclude: true
+scripts:
+- - Visualization, EO Browser, Raw Values
+  - script.js
+examples:
+  evalscripturl: https://custom-scripts.sentinel-hub.com/custom-scripts/sentinel-2/kndvi/script.js
+---
+
+# Evaluate and Visualize
+
+- Interlaken, Switzerland (to show green vegetation, bare rock, snow and ice, and clouds together with kNDVI)
+  - [EO Browser](https://sentinelshare.page.link/kD7v)
+  - [Copernicus Browser](https://link.dataspace.copernicus.eu/dzpu)
```

Comment:
I would suggest following the [example](https://github.com/sentinel-hub/custom-scripts/blob/main/contribute/example-multiple-scripts/index.md#adding-examples) in the contribution markdown.
```
examples:
- zoom: '10'
  lat: '42.76703'
  lng: '11.22847'
  datasetId: S2L2A
  fromTime: '2020-07-12T00:00:00.000Z'
  toTime: '2020-07-12T23:59:59.999Z'
  platform:
  - CDSE
  - EOB
  evalscripturl: https://custom-scripts.sentinel-hub.com/custom-scripts/sentinel-2/ndvi/eob.js
```
The local build looks a bit inconsistent now: 
<img width="769" alt="Screenshot 2024-05-13 at 13 25 34" src="https://github.com/sentinel-hub/custom-scripts/assets/78031577/6f2d1724-965e-4a7f-be08-30aef2feeedd">